### PR TITLE
coinutils 2.11.2 (new formula)

### PR DESCRIPTION
**Pull request by jamiesnape**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Relates #40725. Note in Debian as [coinor-libcoinutils-dev](https://packages.debian.org/buster/coinor-libcoinutils-dev) (also new to GitHub, the project used its own Subversion repos for many years).

### CODE REVIEW

File: Formula/coinutils.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+class Coinutils < Formula
+  desc "COIN-OR utilities"
+  homepage "https://github.com/coin-or/CoinUtils"
+  url "https://github.com/coin-or/CoinUtils/archive/releases/2.11.2.tar.gz"
+  sha256 "30c7f6c84dbb9f6e4fe5bbe4015ed15e2d1402146f8354cfc50c34d8735a49b1"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
```
Comment (fxcoudert) [inline Formula/coinutils.rb]: Why does this need GCC? It's a very heavy dependency, and we prefer to build formulas with the system compiler.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/coinutils.rb (hunk 1 above)
Comment (soonho-tri) [reply Formula/coinutils.rb]: It depends on fortran (`gfortran`): https://github.com/coin-or/CoinUtils/blob/master/CoinUtils/configure.ac#L52-L59

Comment (fxcoudert) [reply Formula/coinutils.rb]: There is not a single line of Fortran in the project, it should be able to compile without it.

Comment (jamiesnape) [reply Formula/coinutils.rb]: Removed

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**